specification/src/chapter3.adoc

```diff
@@ -1,45 +1,313 @@
+
+MMU, PMP/ePMP, and sPMP are discussed later in this chapter.
+
+==== PMP and ePMP
+
+https://github.com/riscv/riscv-isa-manual/releases/tag/Priv-v1.12[Privileged ISA]
+
+_Physical memory protection (PMP)_ enables M-mode to control physical memory access for supervisor or HS mode. 
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN  
+| PMP configurations MUST only be directly accessible to machine mode
+|===
+
+Individual access controlled regions can be locked until the next system reset to create temporal isolation boundaries, such as protecting immutable boot code. 
+
+ePMP extends PMP protection by allowing machine mode to restrict its own access to memory allocated to lower privilege levels. This can be used to mitigate against privilege escalation attacks, for example.
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN  
+| If PMP is supported then ePMP MUST be supported.
+|===
+
+==== sPMP
+
+https://github.com/riscv/riscv-spmp
+
+_Supervisor PMP (sPMP)_ enables supervisor mode to control physical memory access for U mode.
+
+sPMP allows supervisor mode to restrict its own access to memory allocated to lower privilege levels. This can be used to mitigate against privilege escalation attacks, for example.
+
+When combined with H-extension, sPMP can be nested so that the hypervisor can control memory allocations to its guests, and each guest can control its own memory allocations to its workloads.
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN  
+| PMP/ePMP MUST be used to protect M-mode from lower privilege levels.
+|===
+
+==== MMU
+
+https://github.com/riscv/riscv-isa-manual/releases/tag/Priv-v1.12[Privileged ISA]
+
+_Memory management unit (MMU)_ enables memory virtualization for:
+
+* Isolating an OS from workloads on a system without H-extension (one-stage translation)
+* Isolating a hypervisor from a guest, on a system with H-extension (two-stage translation)
+ 
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN  
+| PMP/ePMP MUST be used to protect M-mode from lower privilege levels.
+|===
+
+==== Supervisor domains
+
+https://github.com/riscv/riscv-smmtt
+
+Supervisor domains allow software components on the same hart to be developed, certified, deployed and attested independently of each other.
+
+A supervisor domain is a compartment above M-mode, physically isolated - memory, execution state, and devices - from other supervisor domains regardless of privilege level (above M-mode). Isolation and context switching between supervisor domains are managed by M-mode firmware. 
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN  
+| PMP/ePMP MUST be used to protect machine mode from any supervisor domain.
+
+| CAT_NN
+| Either PMP/ePMP or MTT MUST be used to enforce physical memory isolation boundaries for supervisor domains.
+
```

Comment:
The two should be merged. The second is stronger - either PMP/ePMP or MTT

Reply:
addressed